packages/zoe/test/unitTests/contracts/test-multipoolAutoswap.js

```diff
@@ -1841,3 +1841,32 @@ test('multipoolAutoSwap jig - insufficient', async t => {
   t.deepEqual(await moolaR.issuer.getAmountOf(refund), moola(200n));
   t.deepEqual(await centralR.issuer.getAmountOf(payout), centralTokens(0));
 });
+
+test('multipoolAutoSwap collect empty fees', async t => {
+  const zoe = makeZoe(fakeVatAdmin);
+  const centralR = makeIssuerKit('central');
+  const centralTokens = value => amountMath.make(value, centralR.brand);
+
+  const bundle = await bundleSource(multipoolAutoswapRoot);
+  const installation = await zoe.install(bundle);
+  // This timer is only used to build quotes. Let's make it non-zero
+  const fakeTimer = buildManualTimer(console.log, 30);
+  const { publicFacet, creatorFacet } = await zoe.startInstance(
+    installation,
+    harden({ Central: centralR.issuer }),
+    { timer: fakeTimer },
+  );
+  const aliceCollectFeesInvitation = await E(
+    creatorFacet,
+  ).makeCollectFeesInvitation();
+
+  const feeSeat = await zoe.offer(aliceCollectFeesInvitation);
+  t.deepEqual(await E(feeSeat).getCurrentAllocation(), {});
```

Comment:
This looks good! Can we test getting a payout, if that is an expected part of the fee extraction?

Reply:
added.